src/systems/filecoin_blockchain/chainsync/_index.md

```diff
@@ -345,22 +347,23 @@ State Machine:
 - `ChainSync` SHOULD keep a cache of unvalidated blocks (ideally sorted by likelihood of belonging to the chain), and delete unvalidated blocks when they are passed by `FinalityTipset`, or when `ChainSync` is under significant resource load.
-- It is key to note that any block received after the `ROUND_CUTOFF` time must be automatically discarded by the miner until the start of the next epoch.
+- It is key to note that any block received after the `EPOCH_CUTOFF` time must be automatically discarded by the miner until the start of the next epoch.
 
 - **Progressive Stages of Block Validation**
   - _(TODO: move this to blockchain/Block section)_
   - **BV0 - Syntactic Validation**: Validate data structure packing and ensure correct typing.
-  - **BV1 - Light Consensus State Checks**: Validate `b.ChainWeight`, `b.ChainEpoch`, `b.MinerAddress`, `b.Timestamp`, are plausible (some ranges of bad values can be detected easily, especially if we have the state of the chain at `b.ChainEpoch - consensus.LookbackParameter`. Eg Weight and Epoch have well defined valid ranges, and `b.MinerAddress`
-  must exist in the lookback state). This requires some chain state, enough to establish plausibility levels of each of these values. A node should be able to estimate valid ranges for `b.ChainEpoch` based on the `LastTrustedCheckpoint`. `b.ChainWeight` is easy if some of the relatively recent chain is available, otherwise hard.
-  - **BV2 - Signature Validation**: Verify `b.BlockSig` is correct.
-  - **BV3 - Verify ElectionProof**: Verify `b.ElectionProof` is correct. This requires having state for relevant lookback parameters.
-  - **BV4 - Verify Ancestry links to chain**: Verify ancestry links back to trusted blocks. If the ancestry forks off before finality, or does not connect at all, it is a bad block.
-  - **BV4 - Verify MessageSigs**: Verify the signatures on messages
-  - **BV5 - Verify StateTree**: Verify the application of `b.Parents.Messages()` correctly produces `b.StateTree` and `b.MessageReceipts`
+  - **BV1 - Epoch Window Validation**: Validate that block was produced during the current epoch.  Validate that the block was received before `EPOCH_CUTOFF`.  This stage only applies during `CHAIN_FOLLOW`.
+  - **BV2 - Light Consensus State Checks**: Validate `b.ChainWeight`, `b.ChainEpoch`, `b.MinerAddress`, `b.Timestamp`, are plausible (some ranges of bad values can be detected easily, especially if we have the state of the chain at `b.ChainEpoch - consensus.LookbackParameter`. Eg Weight and Epoch have well defined valid ranges, and `b.MinerAddress`
+  must exist in the lookback state). This requires some chain state, enough to establish plausibility levels of each of these values. A node can determine the exact valid range for `b.ChainEpoch` based on the `LastTrustedCheckpoint`, and a synchronized clock (see {{<sref clock>}}). `b.ChainWeight` is easy if some of the relatively recent chain is available, otherwise hard.
```

Comment:
HELP ACTION REQUIRED: right now this link does not render and idk how to fix.

Reply:
could make the last sentence wrt chainWeight more precise. Eg. "Verifying `b.ChainWeight` will require computation that scales linearly with the length of the chain to verify"

Reply:
must add `clock` to `hugo/data/sref_titles.yaml` and `hugo/data/sref_locations.yaml` in order for the label to be recognized